src/PostCampaign.php

```diff
@@ -29,6 +29,7 @@ class PostCampaign {
 		'campaign_body_font',
 		'campaign_footer_theme',
 		'footer_links_color',
+		'country-selector--countries-list--background',
```

Comment:
This is a consequence of the current way it stores the CSS variables set through the sidebar: each one gets an individual meta field.

We need to add this here because all items of this `const` are registered for the rest API, otherwise they won't be stored, and silently ignored.

We should rewrite this to use a single field that holds a JSON object with all variables, so that we can easily add more.